src/lib/components/explorer/results/ResultsPanel.svelte

```diff
@@ -56,15 +56,31 @@
     triggerRefreshCount = getCount();
   });
 
+  function clearFiltersModal() {
+    modalStore.trigger({
+      type: 'confirm',
+      title: 'Clear All Filters',
+      body: 'Are you sure you want to clear all filters?',
```

Comment:
@emhug Should this indicate that you are also clearing exports?

Reply:
I think this is actually fine for now. I will make a note to get user feedback on this to make sure the language is clear.